src/test/java/seedu/address/logic/commands/MarkCommandTest.java

```diff
@@ -0,0 +1,91 @@
+package seedu.address.logic.commands;
+
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static seedu.address.logic.commands.CommandTestUtil.assertTaskCommandFailure;
+import static seedu.address.logic.commands.CommandTestUtil.assertTaskCommandSuccess;
+import static seedu.address.testutil.Assert.assertThrows;
+import static seedu.address.testutil.TypicalIndexes.INDEX_FIRST;
+import static seedu.address.testutil.TypicalIndexes.INDEX_SECOND;
+import static seedu.address.testutil.TypicalPersons.getTypicalAddressBook;
+import static seedu.address.testutil.TypicalTasks.getTypicalTaskRepository;
+
+import org.junit.jupiter.api.Test;
+
+import seedu.address.commons.core.Messages;
+import seedu.address.commons.core.index.Index;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.ModelManager;
+import seedu.address.model.OfficeConnectModel;
+import seedu.address.model.Repository;
+import seedu.address.model.RepositoryModelManager;
+import seedu.address.model.UserPrefs;
+import seedu.address.model.mapping.PersonTask;
+import seedu.address.model.task.Task;
+import seedu.address.testutil.TaskBuilder;
+
+public class MarkCommandTest {
+    private Model model = new ModelManager(getTypicalAddressBook(), new UserPrefs());
```

Comment:
can consider using a stub for ModelManager with all methods failing (similar to ab3)